.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+For urgent operational issues, please contact AWS Support directly.
+https://aws.amazon.com/premiumsupport/
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!-- Not required for feature requests -->
+### Expected Behavior
+
+
+### Observed Behavior
+
+
+### Environment Details
+<!--
+Examples:
+* docker info
+* curl http://localhost:51678/v1/metadata
+* df -kh
```

Comment:
Unless I'm mistaken, `-k` and `-h` are mutually exclusive. Whichever flag comes last wins (similarly with `--block-size` and the `-H` flag). If what you're wanting is human readable output with a block size of 1k (vs 1024), use `-H`.